examples/nirfmxwlan/ofdm-modacc.py

```diff
@@ -0,0 +1,438 @@
+                instrument=instr, selector_string="", timeout=10.0
+            )
+        )
+    )
+    sigbcrc_status = ofdm_mod_acc_fetch_sigbcrc_status_response.sigb_crc_status
+
+    ofdm_mod_acc_fetch_pilot_constellation_trace_response = raise_if_error(
+        client.OFDMModAccFetchPilotConstellationTrace(
+            nirfmxwlan_types.OFDMModAccFetchPilotConstellationTraceRequest(
+                instrument=instr, selector_string="", timeout=10.0
+            )
+        )
+    )
+    pilot_constellation = ofdm_mod_acc_fetch_pilot_constellation_trace_response.pilot_constellation
+
+    ofdm_mod_acc_fetch_data_constellation_trace_response = raise_if_error(
+        client.OFDMModAccFetchDataConstellationTrace(
+            nirfmxwlan_types.OFDMModAccFetchDataConstellationTraceRequest(
+                instrument=instr, selector_string="", timeout=10.0
+            )
+        )
+    )
+    data_constellation = ofdm_mod_acc_fetch_data_constellation_trace_response.data_constellation
+
+    ofdm_mod_acc_fetch_chain_rmsevm_per_subcarrier_mean_trace_response = raise_if_error(
+        client.OFDMModAccFetchChainRMSEVMPerSubcarrierMeanTrace(
+            nirfmxwlan_types.OFDMModAccFetchChainRMSEVMPerSubcarrierMeanTraceRequest(
+                instrument=instr, selector_string="", timeout=10.0
+            )
+        )
+    )
+    x0 = ofdm_mod_acc_fetch_chain_rmsevm_per_subcarrier_mean_trace_response.x0
+    dx = ofdm_mod_acc_fetch_chain_rmsevm_per_subcarrier_mean_trace_response.dx
+    chain_rmsevm_per_subcarrier_mean = (
+        ofdm_mod_acc_fetch_chain_rmsevm_per_subcarrier_mean_trace_response.chain_rms_evm_per_subcarrier_mean
+    )
+
+    print("------------------EVM------------------\n")
+    print("------------------Composite EVM------------------")
+    print(f"RMS EVM Mean (dB)                       : {composite_rmsevm_mean}")
+    print(f"Data RMS EVM Mean (dB)                  : {composite_data_rmsevm_mean}")
+    print(f"Pilot RMS EVM Mean (dB)                 : {composite_pilot_rmsevm_mean}\n")
+    print(f"Number of Symbols Used                  : {number_of_symbols_used}\n")
+    print("------------------Impairments & PPDU Info------------------\n")
+    print(f"Frequency Error Mean (Hz)               : {frequency_error_mean}")
+    print(f"Symbol Clock Error Mean (ppm)           : {symbol_clock_error_mean}\n")
+    print("------------------IQ Impairments------------------")
+    print(f"Relative I/Q Origin Offset Mean (dB)    : {relative_iq_origin_offset_mean}")
+    print(f"Absolute I/Q Origin Offset Mean (dBm)   : {absolute_iq_origin_offset_mean}")
+    print(f"I/Q Gain Imbalance Mean (dB)            : {iq_gain_imbalance_mean}")
+    print(f"I/Q Quadrature Error Mean (deg)         : {iq_quadrature_error_mean}")
+    print(f"I/Q Timing Skew Mean (s)                : {iq_timing_skew_mean}")
+    print("\n------------------PPDU Info------------------")
+    if ppdu_type == 0:
```

Comment:
All of these are Enums with types defined in the protofile we should use those instead.